components/layout-switcher.tsx

```
@@ -1,113 +1,240 @@
-const layoutMap = {
-  classic: { 
+const getLayoutMap = (isDark: boolean) => ({
+  Home_1: { 
     name: "Home 1", 
     description: "Classic layout with traditional design",
+    color: "blue",
+    icon: <Layout className="w-4 h-4" />,
     preview: (
-      <div className="w-full h-20 bg-gray-100 dark:bg-gray-800 rounded-lg overflow-hidden border border-blue-400 dark:border-blue-400 relative">
+      <div className="w-full h-32 bg-gradient-to-br from-blue-50 to-blue-100 dark:from-blue-950 dark:to-blue-900 rounded-xl overflow-hidden border-2 border-blue-300 dark:border-blue-600 relative group shadow-lg hover:shadow-xl transition-all duration-300">
+        <div className="absolute inset-0 bg-gradient-to-t from-black/20 to-transparent opacity-0 group-hover:opacity-100 transition-opacity duration-300"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)